packages/theme/helpers/product/productData.ts

```diff
@@ -1,21 +1,14 @@
 import { computed, useRoute } from '@nuxtjs/composition-api';
-import { productGetters } from '~/getters';
 
 export const productData = (products) => {
   const route = useRoute();
-  const { params: { id }, query } = route.value;
+  const { params: { id } } = route.value;
 
   return {
     id,
     product: computed(() => {
       if (!products) return {};
-      const baseProduct = Array.isArray(products.value?.items) && products.value?.items[0] ? products.value?.items[0] : [];
-
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-      return productGetters.getFiltered(baseProduct, {
-        master: true,
-        attributes: query,
-      });
+      return Array.isArray(products.value?.items) && products.value?.items[0] ? products.value?.items[0] : [];
```

Comment:
```suggestion
      return products.value?.items?.[0] ?? [];
```
optional, I know it's not your code

Reply:
this could potentially explode if items can be an object though